oshi-core/src/main/java/oshi/hardware/platform/linux/LinuxCentralProcessor.java

```diff
@@ -57,6 +57,21 @@
 
     // See https://www.kernel.org/doc/Documentation/cpu-freq/user-guide.txt
     private static final String CPUFREQ_PATH = "/sys/devices/system/cpu/cpu";
+    
+    private static final long BOOTTIME;
+    static {
+    	// Boot time given by btime variable
+    	// in /proc/stat.
+        List<String> procStat = FileUtil.readFile("/proc/stat");
+        for (String stat : procStat) {
+            if (stat.startsWith("btime")) {
+                String[] bTime = ParseUtil.whitespaces.split(stat);
+                BOOTTIME = ParseUtil.parseLongOrDefault(bTime[1], 0L);
+                break;
+            }
+        }
+    	
```

Comment:
You need to handle the case where the final variable isn't initialized in the loop.

Reply:
Hi @dbwiddis. How do you suggest I handle the case where BOOTTIME isn't initialized in the loop (here and in the solaris and windows cases)? If the record isn't found, in this instance, in the /proc/stat file, is there somewhere else I can find the value? Would you recommend I set the BOOTTIME to 0?

Reply:
I think the current time in seconds (`System.currentTimeMillis/1000`) minus uptime in seconds is a reasonable default.